zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/appliers/GroupAppliersTest.java

```diff
@@ -110,23 +111,22 @@ void shouldAddUserEntityToGroup() {
             .setPassword("password")
             .setEmail("test@example.com");
     userState.create(userRecord);
-    final var groupId = "1";
-    final var groupKey = Long.parseLong(groupId);
+    final var groupId = "123";
     final var groupName = "group";
     final var entityType = EntityType.USER;
-    final var groupRecord =
-        new GroupRecord().setGroupKey(groupKey).setGroupId(groupId).setName(groupName);
+    final var groupRecord = new GroupRecord().setGroupId(groupId).setName(groupName);
     groupState.create(groupRecord);
     groupRecord.setEntityKey(entityKey).setEntityType(entityType);
 
     // when
-    groupEntityAddedApplier.applyState(groupKey, groupRecord);
+    groupEntityAddedApplier.applyState(1L, groupRecord);
 
     // then
-    final var entitiesByType = groupState.getEntitiesByType(groupKey);
+    final var entitiesByType = groupState.getEntitiesByType(groupId);
     assertThat(entitiesByType).containsOnly(Map.entry(entityType, List.of(entityKey)));
-    final var persistedUser = userState.getUser(entityKey).get();
-    assertThat(persistedUser.getGroupKeysList()).containsExactly(groupKey);
+    // TODO: re-enable with https://github.com/camunda/camunda/issues/30091
+    //    final var persistedUser = userState.getUser(entityKey).get();
+    //    assertThat(persistedUser.getGroupKeysList()).containsExactly(groupId);
```

Comment:
🔧 This can just be removed. With https://github.com/camunda/camunda/pull/30529 the persisted user won't have a list of group keys/ids anymore